static/bidder-params/mobkoi.json

```diff
@@ -8,10 +8,10 @@
       "type": "string",
       "description": "Placement ID"
     },
-    "adServerBaseUrl": {
+    "integrationEndpoint": {
```

Comment:
I see we’re removing one of the existing fields here. Will this change be backwards compatible? Could it impact any existing live integrations that rely on this field?